Dockerfile

```diff
@@ -59,8 +59,8 @@ RUN wget https://github.com/karalabe/xgo/blob/master/docker/base/patch.tar.xz?ra
 
 # Setup Freebsd, stolen from https://github.com/sandvine/freebsd-cross-build
 RUN mkdir -p /freebsd/x86_64-pc-freebsd12 && cd /freebsd \
- && wget ftp://ftp-archive.freebsd.org/pub/FreeBSD/releases/amd64/amd64/12.3-RELEASE/base.txz \
- && wget ftp://ftp-archive.freebsd.org/pub/FreeBSD/releases/amd64/amd64/12.3-RELEASE/lib32.txz \
+ && wget http://ftp-archive.freebsd.org/pub/FreeBSD-Archive/old-releases/amd64/12.3-RELEASE/base.txz \
```

Comment:
Is there a https site we could download these?

Reply:
I don't think there is, maybe verification is done with checksum?

Reply:
Verifying the checksum would be fine as well, however I don't see it being verified here.

Reply:
true, I'll get another change opened to add the checksum verification

Reply:
https://github.com/storj/docker-golang/pull/21